specs/interop/predeploys.md

```diff
@@ -276,18 +298,12 @@ The following function is used for sending messages between domains:
 function sendMessage(uint256 _destination, address _target, bytes calldata _message) external;
 ```
 
-It creates an initiating message that is represented by an anonymous event:
+It emits a `SentMessage` event with the necessary metadata to execute when relayed on the destination chain.
 
 ```solidity
-assembly {
```

Comment:
It could be good to include this code snippet under the `executeMessage` section to illustrate how to encode a call directly in an event. Something like

```solidity
bytes memory data = abi.encodeCall(Foo.bar, (1, address(0x20)));
assembly {
  log0(add(data, 0x20), mload(data))
}
```

Reply:
yeah good idea, added this as reference with a note about the L2ToL2CDM